client/src/sections/Fridge/Sidebar/Sidebar.tsx

```diff
@@ -1,11 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
+ 
 
 import './Sidebar.css';
 
 export default function Sidebar() {
+
+  const [categories, setCategories] = useState([
+    "Meats", "Fruits", "Veggies"
+  ])
+
   return (
       <div className='sidebar'>
-            Sidebar placeholder
+        {categories.map( i => (
```

Comment:
Lets move the logic out of the render and make a constant on top for the markup.

```
const categoriesMarkup = categories.map((category, i) => {
    return (
      <div className="category" key={i}>{category}</div>
    );
  })

return (
    <div className='sidebar'>
       {categoriesMarkup}
       <div className="category" onClick={() => setCategories(categories => [...categories, "apple"])}>+</div>
    </div>
);
```